src/main/java/frc/robot/Robot.java

```diff
@@ -52,21 +46,16 @@ public void robotInit() {
         rootNamespace.putBoolean("digital input 4", digitalInput.get());
         rootNamespace.putData("intake cargo", new IntakeCargo());
         rootNamespace.putBoolean("transfer limit", transfer::getStrapEntranceSensor);
-        rootNamespace.putData("test intake", new ParallelCommandGroup(
-                new MoveGenericSubsystem(intakeRoller, IntakeRoller.MAX_SPEED) {
-                    @Override
-                    public boolean isFinished() {
-                        return intakeToTransfer.getLimit();
-                    }
-                },
-                new MoveGenericSubsystem(intakeToTransfer, IntakeToTransfer.SPEED) {
-                    @Override
-                    public boolean isFinished() {
-                        return transfer.getStrapEntranceSensor();
-                    }
+        intakePlacer.setDefaultCommand(new MoveGenericSubsystem(intakePlacer, 0.2) {
```

Comment:
Can you figure out what's wrong here on your own?

Reply:
Magic number, but that doesn't matter since Yotam has another pull request to change it anyways (so no need to change it here too)